### PR TITLE
fix: replace group_uuid with enterprise_customer_uuid

### PR DESCRIPTION
**Pull request by muhammad-ammar**

### CODE REVIEW

File: enterprise_access/apps/api/v1/tests/test_views.py [hunk 1]
```diff
@@ -1711,7 +1711,7 @@ def test_list_policies_for_enterprise(self):
 
     def test_list_without_group_uuid_query_param(self):
```
Comment (sameenfatima78) [inline enterprise_access/apps/api/v1/tests/test_views.py:1712]: test name had group_uuid in it, can we replace it with enterprise_uuid?

**Review by sameenfatima78 — APPROVED** [review]
one minor change requested, otherwise all good!

File: enterprise_access/apps/api/v1/tests/test_views.py (hunk 1 above)
Comment (muhammad-ammar) [reply enterprise_access/apps/api/v1/tests/test_views.py:1712]: good catch.

Comment (muhammad-ammar) [reply enterprise_access/apps/api/v1/tests/test_views.py]: done

**Pull request merged**